src/main/java/betterquesting/api2/client/gui/themes/presets/PresetIcon.java

```diff
@@ -99,7 +100,8 @@ public enum PresetIcon {
     ICON_MENU("icon_menu"),
 
     ICON_PATREON("icon_patreon"),
-    ICON_TWITCH("icon_twitch");
+    ICON_TWITCH("icon_twitch"),
+    ICON_LOADING("icon_refresh");
```

Comment:
this has to have a different name, as `icon_refresh` already exists and is used as the `Reset Quest` icon on the designer. this change causes it to be constantly rotating.